recipes/jerryscript/all/conanfile.py

```diff
@@ -14,15 +16,82 @@ class JerryScriptStackConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "tool_cmdline": [True, False],
+        "tool_cmdline_test": [True, False],
+        "tool_cmdline_snapshot": [True, False],
+        "libfuzzer_support": [True, False],
+        "default_port_implementation": [True, False],
+        "jerry_ext": [True, False],
+        "jerry_math": [True, False],
+        "link_time_optimization": [True, False],
+        "strip_symbols": [True, False],
+        "amalgamated": [True, False],
+        "debugger": [True, False],
+        "keep_line_info": [True, False],
+        "profile": "ANY",
+        "promise_callback": [True, False],
+        "external_context": [True, False],
+        "snapshot_execution": [True, False],
+        "snapshot_saving": [True, False],
+        "parser": [True, False],
+        "enable_dump_bytecode": [True, False],
+        "enable_dump_regexp_bytecode": [True, False],
+        "strict_regexp": [True, False],
+        "error_messages": [True, False],
+        "logging": [True, False],
+        "memory_statistics": [True, False],
+        "heap_size": "ANY",
+        "gc_limit": range(8193),
```

Comment:
Is a gc_limit > 8193 impossible
How about making this just "ANY" and asserting in `validate` that the value is an integer?
e.g.
```python
def validate(self):
    try:
        v = int(self.options.gc_limit)
        if v <= 0:
            raise ValueError("gc_limit must be positive")
    except ValueError:
        raise ConanInvalidConfiguration("gc_limit must be a positive integer.
```